web/webkit/src/main/scala/net/liftweb/http/HtmlNormalizer.scala

```diff
@@ -0,0 +1,291 @@
+package net.liftweb
+package http
+
+import scala.xml._
+import scala.collection.immutable.Vector
+
+import js.JsCmd
+import js.JsCmds.Noop
+import js.JE.{AnonFunc, Call, JsRaw}
+
+import util.Helpers._
+
+/**
+ * Represents an HTML attribute for an event handler. Carries the event name and
+ * the JS that should run when that event is triggered as a String.
+ */
+private case class EventAttribute(eventName: String, jsString: String)
```

Comment:
Is this an appropriate point to have Javascript as a `String` instead of as a `JValue`?

Not saying it isn't, mind, I just don't know this code well enough to know myself and it always seems like a question worth asking. :-)